cli/features/support/env.ls

```diff
@@ -13,3 +14,12 @@ module.exports = ->
     if @app-dir
       fs.remove-sync @app-dir
     kill-child-processes done
+
+
+  #stop and remove all running docker containers
+  @After tags: ['~@docker-cleanup'], timeout: 20_000, (scenario, done) ->
+    running-containers = child_process.exec-sync 'docker ps -q' |> (.to-string!)
```

Comment:
Can you add one? It would be good if all Docker-related functionality is encapsulated in this class, rather than spread all over the codebase.